packages/trip-form/src/MetroModeSelector/MetroModeSelector.story.tsx

```diff
@@ -69,19 +69,12 @@ const defaultModeButtonDefinitions = [
 
 const modeSettingDefinitionsWithDropdown = [
   ...modeSettingDefinitions,
-  {
-    applicableMode: "TRANSIT",
-    default: "blue",
-    key: "busColor",
-    label: "Bus Color",
-    options: [{ value: "blue", text: "Blue" }],
-    type: "DROPDOWN"
-  },
```

Comment:
Can we keep the dropdown? I know it's broken but it's nice to have as a reference. I'd like to bring it back as an option to replace the sliders at some point